packages-nextgen/kestrel_core/src/kestrel/frontend/compile.py

```diff
@@ -88,8 +133,13 @@ def assignment(self, args):
 
     def get(self, args):
         graph = IRGraph()
+        entity_name = args[0].value
+        mapped_entity_name = self.entity_map.get(entity_name, entity_name)
+        filter = args[2]
```

Comment:
Might want to avoid the variable name `filter` since it's a global function in Python

Reply:
Addressed in commit https://github.com/opencybersecurityalliance/kestrel-lang/pull/434/commits/1a375dc2d5bb98b706c806624e1552c20a660b12